hardware/targets.json

```diff
@@ -548,5 +548,34 @@
                 "platform": "esp8285"
             }
         }
+    },
+    "mfd-crossbow": {
+        "name": "MFD Crossbow Antenna Tracker Backpack",
+        "vrx": {
```

Comment:
I think this should be "aat" so we can lump all the antenna trackers in the same section.

Reply:
Sure, we can do that. Its a more fitting tag...
In saying that, I don't think it actually changes anything at all, except for that json field in the targets file (or at least I couldn't see anything in the build files that this would influence).

Reply:
Oh, but it will in the new web-flasher I'm working on.